### PR TITLE
SPK openHAB 3.1.0

### PR DESCRIPTION
**Pull request by tknaller**
SPK openHAB 3.1.0, replace bintray with jfrog

---
This change is [Reviewable](https://reviewable.io/reviews/openhab/openhab-syno-spk/188)

### CODE REVIEW

**Review by tknaller — COMMENTED** [review]
> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/openhab/openhab-syno-spk/188)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @tknaller)

File: README.md [hunk 1]
```diff
@@ -4,11 +4,20 @@ This project build a Synology DiskStation SPK install package for [openHAB](http
...
+Install .spk via Synology Package manager.
+
+
 ## Installation script behaviour
 
-The Current version of installer `3.0.0` checks most stable recent version of OpenHAB and installs it during the installation procedure.
+The Current version of installer `3.1.0` checks most stable recent version of OpenHAB and installs it during the installation procedure.
 
-You need Java 11 to be able to run openHAB >= 3.0.0, so please download AdoptOpenJDK for [arm32](https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.9.1%2B1/OpenJDK11U-jdk_aarch64_linux_hotspot_11.0.9.1_1.tar.gz) because Oracle is not offering 32bit compatible package.
+You need Java 11 to be able to run openHAB >= 3.1.0, so please download AdoptOpenJDK for [arm32](https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.9.1%2B1/OpenJDK11U-jdk_aarch64_linux_hotspot_11.0.9.1_1.tar.gz) because Oracle is not offering 32bit compatible package.
```
Comment (eporsche) [inline README.md:20]: For 214play and other 32bit based intel systems you can download a working java11 version from https://bell-sw.com/pages/downloads/#/java-11-lts

**Review by tknaller — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/openhab/openhab-syno-spk/188)* status: 2 of 3 files reviewed, 1 unresolved discussion (waiting on @eporsche, @tknaller, and @Toover)

___
*[README.md, line 20 at r2](https://reviewable.io/reviews/openhab/openhab-syno-spk/188#-MduMpWN7hx-Mw5Mph3Y-r2-20:-Mp6anBHB4bMavM3ITbT:b-gzeb0o) ([raw file](https://github.com/openhab/openhab-syno-spk/blob/7c5c94ff6358d713c2a62de43ce17d959530419a/README.md#L20)):*
<details><summary><i>Previously, eporsche wrote…</i></summary><blockquote>

For 214play and other 32bit based intel systems you can download a working java11 version from https://bell-sw.com/pages/downloads/#/java-11-lts
</blockquote></details>

updated text. getting the feeling that contributing a little fix made me maintainer...

**Review by tknaller — COMMENTED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/openhab/openhab-syno-spk/188)* status: all files reviewed, 1 unresolved discussion (waiting on @eporsche)

**Comment by eporsche** [comment]
> > _[Reviewable](https://reviewable.io/reviews/openhab/openhab-syno-spk/188)_ status: 2 of 3 files reviewed, 1 unresolved discussion (waiting on @eporsche, @tknaller, and @Toover)
> 
> _[README.md, line 20 at r2](https://reviewable.io/reviews/openhab/openhab-syno-spk/188#-MduMpWN7hx-Mw5Mph3Y-r2-20:-Mp6anBHB4bMavM3ITbT:b-gzeb0o) ([raw file](https://github.com/openhab/openhab-syno-spk/blob/7c5c94ff6358d713c2a62de43ce17d959530419a/README.md#L20)):_
> Previously, eporsche wrote…
> 
> > For 214play and other 32bit based intel systems you can download a working java11 version from https://bell-sw.com/pages/downloads/#/java-11-lts
> 
> updated text. getting the feeling that contributing a little fix made me maintainer...

^^ ..hope to see this getting merged. I created an spk from your change and it works great on my DS214play with DSM 6!

**Review by tknaller — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/openhab/openhab-syno-spk/188#-:-MpAa77T8uNtVnnm9fTu:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @eporsche)

**Review by cniweb — APPROVED** [review]
:lgtm:

> Reviewed 2 of 3 files at r1, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/openhab/openhab-syno-spk/188#-:-MvVfYuX6mP9Hc4GlXEv:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @eporsche)

**Review by cniweb — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/openhab/openhab-syno-spk/188)* status: all files reviewed, 1 unresolved discussion (waiting on @eporsche)

**Pull request merged**